### PR TITLE
Ignore other login options

### PR DESCRIPTION
**Pull request by JanoschSchaefer**
Ignore other Keycloak login options. This will at least allow you to log in.

Relates to #1078

### CODE REVIEW

File: pkg/provider/keycloak/keycloak.go [hunk 1]
```diff
@@ -393,6 +393,8 @@ func updateKeyCloakFormData(authForm url.Values, s *goquery.Selection, user *cre
 		authForm.Add(name, user.Username)
 	} else if strings.Contains(lname, "password") {
 		authForm.Add(name, user.Password)
+	} else if strings.Contains(lname, "tryanotherway") {
```
Comment (mapkon) [inline pkg/provider/keycloak/keycloak.go:396]: If the intention is to ignore the case, maybe we should log that as the intention?

**Review by mapkon — COMMENTED** [review]
See my comment

File: pkg/provider/keycloak/keycloak.go (hunk 1 above)
Comment (JanoschSchaefer) [reply pkg/provider/keycloak/keycloak.go:396]: I think logging the information will produce more noice than provide any relevant information to the user. Therefore i haven't added any handling besides ignoring this case.

Comment (yeoldegrove) [reply pkg/provider/keycloak/keycloak.go:396]: If the "Try another Way" button is present, the keycloak admin has configured other methods to login.
These are effectively ignored when adding this code, which is totally fine for now (in my opinion).
Nevertheless, the ultimate goal should be adding the possibility to use the other login methods.
Therefore, a small message like: "Ignoring other ways to log in (not implemented)" would be great... Also for the next person who has to debug this ;)

Comment (JanoschSchaefer) [reply pkg/provider/keycloak/keycloak.go:399]: Added debug log that other options are currently not implemented

**Review by mapkon — COMMENTED** [review]
Looks good! @JanoschSchaefer / @yeoldegrove - I am happy to merge this if you are?

**Pull request merged**